### ISSUE TITLE
Add ability to page / jump to letter

### ISSUE THREAD

**Issue by boralyl** [mmatyas/pegasus-frontend#41]
I tried out the latest automated build for rpi3 and cam across a missing usability features that would make nice enhancements.

If you have a large list of games, it takes quite a while to scroll through the list.  It appears R2/L2 are mapped to switch between systems.  It would be nice to use L1/R1 for paging up/down in the game list view like it does in emulation station.

Also it would be nice to have similar functionality in  emulation station where you can jump to a letter.

Finally, it would be nice if when you exited a game that you would be back to the system with the game you were playing highlighted.  Currently if you are playing a game that begins with a `Z` and exit, you will be all the way back at the top of the game list when the front end returns.

**Comment by AndersHP84** [comment]
Reeeeally missing the page down function as well :)

**Comment by mmatyas** [comment]
Page up/down has just been added and is mapped to L2/PageUp and R2/PageDown.

Remembering the launched game was fixed earlier, I just forgot to update this issue (sorry).

All that's left is the letter jump feature.

**Comment by mmatyas** [comment]
Letter jump is now available via Alt + [letter], but there's no UI for it yet.

**Comment by msheehan79** [comment]
Did the letter jump feature get removed or broken at some point? 

I agree for large collections this would be a really helpful feature which I always used in EmulationStation. I just saw this post and tried the "Alt + Letter" on my keyboard but it did not seem to work.

**Comment by mmatyas** [comment]
Yes, the letter jump was temporarily removed after a large set of changes, then I started working on other parts and in the end it wasn't re-added. Will see if I can update it when I have some time.

**Comment by retrorepair** [comment]
Any chance this could be readded? In a complete console set it takes forever to navigate. That or a more robust scrolling action with touchscreens.
